runatlantis.io/.vuepress/config.js

```diff
@@ -99,6 +95,13 @@ module.exports = {
                         'security'
                     ]
                 },
+                {
+                    title: 'Streaming Atlantis logs',
```

Comment:
```suggestion
                    title: 'Real-time Terraform Logs',
```

It's important we don't confuse people since Atlantis logs are technically different.